index.html

```diff
@@ -0,0 +1,258 @@
+                    <a href="#" class="header-links__item">Как заказать</a>
+                    <a href="#" class="header-links__item">Контакты</a>
+                </div>
+            </div>
+        </nav>
+        <div class="header__number">
+            +7 906 899 92 96
+        </div>
+    </header>
+    <main>
+        <section class="base-section main-section">
+            <div class="main-section-info">
+                <h1 class="main-section-info__title">
+                    Торты на заказ
+                </h1>
+                <h3 class="main-section-info__hand-made">
+                    Ручная работа
+                </h3>
+                <button class="common__button main-section-info__button">
+                    Выбрать торт
+                </button>
+                <div class="main-section-info-benefits">
+                    <div class="main-section-info-benefits-item">
+                        <img src="Assets/Styles/site/main-section/main-section-info/main-section-info-benefits/images/Ingredients-icon.png" alt="Natural Ingredients" class="main-section-info-benefits-item__image" />
+                        <p class="main-section-info-benefits-item__text">
+                            Натуральные ингредиенты
+                        </p>
+                    </div>
+                    <div class="main-section-info-benefits-item">
+                        <img src="Assets/Styles/site/main-section/main-section-info/main-section-info-benefits/images/Delivery-icon.png" alt="Delivery" class="main-section-info-benefits-item__image" />
+                        <p class="main-section-info-benefits-item__text">
+                            Доставка бесплатно при заказе от 2000 рублей*
+                        </p>
+                    </div>
+                </div>
+                <button class="common__button main-section-info__button main-section-info__button_screen_mobile">
+                    Выбрать торт
+                </button>
+            </div>
+            <img src="Assets/Styles/site/main-section/images/Illustration1.png" alt="" class="bg-image main-section__bg-illustration1" />
+            <img src="Assets/Styles/site/main-section/images/Illustration2.png" alt="" class="bg-image main-section__bg-illustration2" />
+            <img src="Assets/Styles/site/main-section/images/Illustration3.png" alt="" class="bg-image main-section__bg-illustration3" />
+            <img src="Assets/Styles/site/main-section/images/Cake.png" alt="" class="bg-image main-section__bg-cake" />
+            <img src="Assets/Styles/site/main-section/images/cake-mobile.png" alt="" class="bg-image main-section__bg-cake_screen_mobile" />
+        </section>
+        <section class="base-section our-cakes-section">
+            <h2 class="our-cakes-section__title">
+                Наши торты
+            </h2>
+            <div class="our-cakes-section-types">
+                <div class="our-cakes-section-types-block">
```

Comment:
you use `our-cakes-section-types-block` but then you place here `cake-platforms` , it's a little bit unclear could you please think about the more proper name?